src/jdk.compiler/share/man/javac.md

```diff
@@ -562,12 +561,12 @@ file system locations may be directories, JAR files or JMOD files.
     warnings is recommended.
 
 <a id="option-Xlint-custom">`-Xlint:`\[`-`\]*key*(`,`\[`-`\]*key*)\*</a>
-:   Supplies warnings to enable or disable, separated by comma. Precede a key
-    by a hyphen (`-`) to disable the specified warning.
+:   Specifies warning categories to enable or disable, separated by comma.
```

Comment:
This text uses `categories` and `key` in a very loose way -- they seem to refer to the same concept, yet none seems to be defined.

```
Enables or disables selected warning categories. Each warning category is identified by a key. Some keys allow multiple categories to be enabled or disabled in bulk. Keys are separated by a comma. Precede a key by a hyphen (-) to disable the corresponding warning category.

Supported keys are:
...
```

What do you think?

Reply:
Yes more words are helpful. I think we also need to say that `all` and `none` can't be preceded by a hyphen (I just learned this not long ago :)

What do you think about this?
```
<a id="option-Xlint-custom">`-Xlint:`\[`-`\]*key*(`,`\[`-`\]*key*)\*</a>
:   Enables and/or disables selected warning categories. Categories are identified by one or more
    keys separated by commas. Keys preceded by a hyphen (`-`) disable the corresponding category;
    hyphens are not supported by the special keys `all` and `none`.
```

Reply:
It's good --- but I guess I liked to say that a key was either a warning category (serial) or something to enable/disable things in bulk (all/none). This is now missing in the new text.

Reply:
Hmm how about this:
```
<a id="option-Xlint-custom">`-Xlint:`\[`-`\]*key*(`,`\[`-`\]*key*)\*</a>
:   Enables and/or disables warning categories using the one or more of the keys described
    below separated by commas. The keys `all` and `none` enable or disable all categories
    (respectively); other keys enable the corresponding category, or disable it if preceded
    by a hyphen (`-`).
```

Reply:
I think this looks good -- defo better than current the text it replaces.

Reply:
Great, thanks for the suggestions. Updated in 41ee595d8a1.

I will also update the CSR, please take a look at that too if you have time.

Reply:
> Great, thanks for the suggestions. Updated in [41ee595](https://github.com/openjdk/jdk/commit/41ee595d8a1213a097256899ac3fa10e2a1ad4ae).
> 
> I will also update the CSR, please take a look at that too if you have time.

I've reviewed the CSR -- thanks.